coil-core/src/nonAndroidMain/kotlin/coil3/Image.kt

```diff
@@ -18,33 +19,49 @@ actual interface Image {
     actual val height: Int
     actual val shareable: Boolean
 
-    fun asBitmap(): Bitmap
-}
+    fun Canvas.onDraw()
```

Comment:
Would it be possible for us to make this a common function for `Image`? I'm thinking we could `typealias Canvas = android.graphics.Canvas` on Android and `typealias Canvas = org.jetbrains.skia.Canvas` on non-Android.